eZ/Publish/Core/Persistence/Cache/UrlWildcardHandler.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace eZ\Publish\Core\Persistence\Cache;
+
+use eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler as UrlWildcardHandlerInterface;
+
+class UrlWildcardHandler extends AbstractHandler implements UrlWildcardHandlerInterface
+{
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::create
+     */
+    public function create($sourceUrl, $destinationUrl, $forward = false)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'sourceUrl' => $sourceUrl,
+                'destinationUrl' => $destinationUrl,
+                'forward' => $forward,
+            ]
+        );
+
+        $urlWildcard = $this->persistenceHandler->urlWildcardHandler()->create($sourceUrl, $destinationUrl, $forward);
+
+        // need to clear lists of UrlWildcards cached due to loadAll()
+        $this->cache->invalidateTags(['ez-urlWildcard-all']);
+
+        return $urlWildcard;
+    }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\UrlWildcard\Handler::remove
+     */
+    public function remove($id)
+    {
+        $this->logger->logCall(
+            __METHOD__,
+            [
+                'id' => $id,
+            ]
+        );
```

Comment:
more of a personal preference, but when this fits on one line. But that said maybe more for @alongosz and others to have a say on what they prefer here, or check with symfony CS which we try to align with.

same below

Reply:
I've cleaned it up as you requested.